ext/sqlite3/sqlite3.c

```diff
@@ -2444,6 +2547,49 @@ PHP_MINIT_FUNCTION(sqlite3)
 	REGISTER_LONG_CONSTANT("SQLITE3_OPEN_READWRITE", SQLITE_OPEN_READWRITE, CONST_CS | CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("SQLITE3_OPEN_CREATE", SQLITE_OPEN_CREATE, CONST_CS | CONST_PERSISTENT);
 
+	/* Class constants */
+	zend_declare_class_constant_long(php_sqlite3_sc_entry, "OK", sizeof("OK") - 1, SQLITE_OK);
```

Comment:
I like that you've chosen class constants, but some may see this as inconsistency (all other constants are global).

Reply:
I'm planning to propose a RFC to put all SQLite3 constants as class constants, and deprecate globals constants but with no plan to remove the global constants, as I don't see the point in making a BC-break change just for the sake of making better-looking code. So in the end it should be consistent ;)

Reply:
Fair enough.